rtnetlink/src/addr/add.rs

```diff
@@ -68,17 +70,28 @@ impl AddressAddRequest {
                 message.nlas.push(Nla::Broadcast(brd.octets().to_vec()));
             };
         }
-        AddressAddRequest { handle, message }
+        AddressAddRequest {
+            handle,
+            message,
+            replace: false,
+        }
+    }
+
+    /// Replace existing matching address.
+    pub fn replace(self, replace: bool) -> Self {
```

Comment:
I don't know if we really need a boolean here. Are there cases where we'll call this with `false`? It seems to me that we'll either do `.replace(true)` or not call this method at all.

Reply:
Yep you're right, I tend to leave too much flexibility... Changing this in all `Add*Request`.